R/utils.R

```diff
@@ -81,3 +81,27 @@ ds_is_gh <- function(venue) {
 }
 
 pandoc2.0 <- function() rmarkdown::pandoc_available("2.0")
+
+enfence <- function(path,
+                    tag = NULL,
+                    fallback = "-- nothing to show --") {
+  lines <- readLines(path, encoding = "UTF-8")
+  if (length(lines) == 0) {
+    lines <- fallback
+  }
+  lines <- paste0(c(tag, "``` sh", lines, "```"), collapse = "\n")
+  writeLines(lines, path)
+  path
+}
+
+inject_file <- function(path, inject_path) {
+  lines <- readLines(path, encoding = "UTF-8")
+  lines <- sub(paste0("`", inject_path, "`"), "{{{inject_lines}}}", lines)
```

Comment:
Why not insert the file directly? Why the extra whisker round? 
Also, maybe `fixed = TRUE` is appropriate here.

Reply:
whisker struck me as an easy way to insert it in the right place but yeah I could eliminate that.

Point taken re: `fixed = TRUE`.